assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -104,6 +104,10 @@ function BannerNotification( {
 	title,
 	type,
 	WinImageSVG,
+	smWinImageSVGWidth = 75,
+	smWinImageSVGHeight = 75,
+	mdWinImageSVGWidth = 105,
+	mdWinImageSVGHeight = 105,
```

Comment:
Let's use full words and not these shorthands; they're harder to read, especially without vowels.

```suggestion
	smallWinImageSVGWidth = 75,
	smallWinImageSVGHeight = 75,
	mediumWinImageSVGWidth = 105,
	mediumWinImageSVGHeight = 105,
```

(Same throughout the rest of the file.)